exercises/practice/parallel-letter-frequency/.docs/instructions.append.md

```diff
@@ -0,0 +1,9 @@
+# Instructions append
+
+## Using Parallelism
+
+The goal of this exercise is to practice parallelism with Emacs Lisp.
+
+In Emacs Lisp this is achived by using [`asynchronous processes`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Asynchronous-Processes.html#:~:text=An%20asynchronous%20process%20is%20controlled,%2Dtype%20(see%20below)).
```

Comment:
```suggestion
In Emacs Lisp this can be achieved by using [`asynchronous processes`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Asynchronous-Processes.html#:~:text=An%20asynchronous%20process%20is%20controlled,%2Dtype%20(see%20below)).
```

Small typo fix and grammar fix so we're not saying this is the only way to do parallelism.

Reply:
done